apparmor.d/groups/apparmor/apparmor_parser

```diff
@@ -20,6 +23,7 @@ profile apparmor_parser @{exec_path} flags=(attach_disconnected) {
   @{lib_dirs}/@{multiarch}/** mr,
   @{lib_dirs}/snapd/apparmor.d/{,**} r,
   @{lib_dirs}/snapd/apparmor/{,**} r,
+  /** r, # Can load rules from anywhere
```

Comment:
I disagree on this. Giving access to apparmor_parser is an equivalent to giving mac_admin caps, such only allowing it to load profiles from "trusted" sources is a good thing.

However, it raises the question of development profile, I personally have:
```
  @{user_projects_dirs}/** r,
```
In my local addition to this profile. But that does not apply to everyone and thus, it should be behind a condition.